test/e2e-go/cli/goal/expect/tealAndStatefulTealTest.exp

```diff
@@ -0,0 +1,171 @@
+
+    # Print the transaction id
+    puts "TRANSACTION_ID 1: $TRANSACTION_ID"
+
+    # Check to make sure that the transfer is reflected in the to account
+    ::AlgorandGoal::WaitForAccountBalance $WALLET_1_NAME $ACCOUNT_1_ADDRESS $TRANSFER_AMOUNT $TEST_PRIMARY_NODE_DIR
+
+    # Transfer Algos from primary account to account 2
+    set TRANSACTION_ID [::AlgorandGoal::AccountTransfer $PRIMARY_WALLET_NAME "" $PRIMARY_ACCOUNT_ADDRESS $TRANSFER_AMOUNT $ACCOUNT_2_ADDRESS $FEE_AMOUNT $TEST_PRIMARY_NODE_DIR]
+
+    # Print the transaction id
+    puts "TRANSACTION_ID 2: $TRANSACTION_ID"
+
+    # Check to make sure that the transfer is reflected in the to account
+    ::AlgorandGoal::WaitForAccountBalance $WALLET_2_NAME $ACCOUNT_2_ADDRESS $TRANSFER_AMOUNT $TEST_PRIMARY_NODE_DIR
+
+    # -------------------------- asset setup  ----------------------------------
+
+    set TEAL_SOURCE "$TEST_ROOT_DIR/simple.teal"
+
+    puts "\nwriting teal script to file '$TEAL_SOURCE'"
+
+    set CHAN [open $TEAL_SOURCE w]
+    puts $CHAN "#pragma version 2\nint 1\n"
+    close $CHAN
+
+    puts "reading from file $TEAL_SOURCE"
+    puts [exec cat $TEAL_SOURCE]
+
+    # compile teal assembly to bytecode
+    set APP_ACCOUNT_ADDRESS [::AlgorandGoal::TealCompile $TEAL_SOURCE]
+
+    # -------------------------- create mixed teal and stateful teal transaction  ----------------------------------
+
+    set TRANSACTION_ID_APP [::AlgorandGoal::AccountTransfer $PRIMARY_WALLET_NAME "" $PRIMARY_ACCOUNT_ADDRESS $TRANSFER_AMOUNT $APP_ACCOUNT_ADDRESS $FEE_AMOUNT $TEST_PRIMARY_NODE_DIR]
+
+    # Print the transaction id
+    puts "TRANSACTION_ID_APP: $TRANSACTION_ID_APP, APP_ACCOUNT_ADDRESS: $APP_ACCOUNT_ADDRESS"
+
+    # Check to make sure that the transfer is reflected in the to account
+    ::AlgorandGoal::WaitForAccountBalance $WALLET_1_NAME $APP_ACCOUNT_ADDRESS $TRANSFER_AMOUNT $TEST_PRIMARY_NODE_DIR
+
+    puts "calling app create"
+    set GLOBAL_BYTE_SLICES 1
+    set LOCAL_BYTE_SLICES 0
+    set APP_ID [::AlgorandGoal::AppCreate0 $PRIMARY_WALLET_NAME "" $PRIMARY_ACCOUNT_ADDRESS $TEAL_SOURCE $GLOBAL_BYTE_SLICES $LOCAL_BYTE_SLICES $TEAL_SOURCE $TEST_PRIMARY_NODE_DIR]
+
+    spawn goal app call --app-id $APP_ID --from $ACCOUNT_1_ADDRESS  --out=unsginedtransaction1.tx -d $TEST_PRIMARY_NODE_DIR
+    expect {
+        eof
+    }
+
+    spawn goal clerk send --to=$ACCOUNT_1_ADDRESS --close-to=$ACCOUNT_1_ADDRESS --from-program=$TEAL_SOURCE --amount=409000 --out=unsginedtransaction2.tx -d $TEST_PRIMARY_NODE_DIR
+    expect {
+        eof
+    }
+
+    ::AlgorandGoal::SignTransaction $WALLET_1_NAME $WALLET_1_PASSWORD unsginedtransaction1.tx sginedtransaction1.tx $TEST_PRIMARY_NODE_DIR
+
+    ::AlgorandGoal::InspectTransactionFile unsginedtransaction1.tx
+
+    ::AlgorandGoal::InspectTransactionFile unsginedtransaction2.tx
+
```

Comment:
not sure we want these exceed print outs during regular execution. Maybe move under some debugging condition or remove

Reply:
dropped inspect statements.